### PR TITLE
Docs/bdr/copy edit

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

**Comment by drothery-edb** [comment]
@PJMODOS , Betsy did a copy edit on the first five chapters. If they look ok to you, let's merge them to our branch for the release. She can create a fresh branch next week off of develop for the rest of her edits.

File: product_docs/docs/bdr/4/appusage.mdx [hunk 1]
```diff
@@ -1,130 +1,124 @@
...
 
 BDR supports replicating changes made on one node to other nodes.
 
-BDR will, by default, replicate all changes from INSERTs, UPDATEs, DELETEs
-and TRUNCATEs from the source node to other nodes. Only the final changes
-will be sent, after all triggers and rules have been processed. For example,
-INSERT ... ON CONFLICT UPDATE will send either an INSERT or an UPDATE
-depending on what occurred on the origin. If an UPDATE or DELETE affects
-zero rows, then no changes will be sent.
+BDRs, by default, replicate all changes from INSERT, UPDATE, DELETE
```
Comment (PJMODOS) [inline product_docs/docs/bdr/4/appusage.mdx:11]: There is no BDRs, BDR should always be used as singular. One can say "BDR nodes", but not BDRs.

File: product_docs/docs/bdr/4/camo.mdx [hunk 2]
```diff
@@ -1,108 +1,102 @@
 ---
-navTitle: Commit at Most Once
-title: Commit At Most Once
+navTitle: Commit at most once
+title: Commit at most once
```
Comment (PJMODOS) [inline product_docs/docs/bdr/4/camo.mdx:3]: I don't agree with lower-case here given the feature is literally named "Commit at Most Once (CAMO)" and not "commit at most once"

Comment (PJMODOS) [reply product_docs/docs/bdr/4/camo.mdx:3]: I.e. it's not description, it's a name of an algorithm, just like Eager Replication or Group Commit

File: product_docs/docs/bdr/4/camo.mdx [hunk 3]
```diff
@@ -1,108 +1,102 @@
...
+
+We don't recommend enabling CAMO at the server level, as this imposes
 higher latency for all transactions, even when not needed. Instead,
-we recommend to selectively enable it just for individual transactions
-by turning on CAMO at session or transaction level.
+selectively enable it for individual transactions
+by turning on CAMO at the session or transaction level.
 
-To enable at session level, issue:
+To enable at CAME at the session level:
```
Comment (PJMODOS) [inline product_docs/docs/bdr/4/camo.mdx:93]: CAMO

File: product_docs/docs/bdr/4/camo.mdx [hunk 4]
```diff
@@ -541,76 +527,73 @@ these, CAMO is automatically disabled internally:
...
 
-Proper use of CAMO on the client side is not trivial; we strongly
-recommend testing the application behavior in combination with the BDR
+Proper use of CAMO on the client side isn't trivial. We strongly
+recommend testing the application behavior with the BDR
 cluster against failure scenarios such as node crashes or network
 outages.
 
-## CAMO versus Group Commit
+## CAMO versus group commit
```
Comment (PJMODOS) [inline product_docs/docs/bdr/4/camo.mdx:596]: "Group Commit" is name of a feature/algorithm, don't lowercase please.

**Comment by PJMODOS** [comment]
There are some comments I had, it also bit-rotted as some content from CAMO/CLCD moved to Known Issues so it needs another pass.

General comment I have is that we'll need to synchronize on capitalization of things. I.e. either we keep the feature/algorithm names that are source of acronyms lower case everywhere or mixed case everywhere but currently it's rather mixed and the copy editing made it even more mixes:
- CAMO - Commit At Most Once is BDR specific algorithm, not description of what it does
- Eager - Eager Replication is BDR specific algorithm, not description of what it does
- CLCD - Column Level Conflict Detection, this is more complex as it's both what it does and algorithm, so probably lowercase is justified similar to CRDT
- RLC - Replication Lag Control or Lag Control  - again BDR specific algorithm, partially also what it does, but what it does is actually write throttling

Either we should keep these all lower-case or mixed case but currently this lower-cased all mentions of Commit At Most Once but kept Eager Replication mixed case for example. I don't really mind which we chose, but we should chose consistently.

**Pull request merged**